ci/taos/checker-pr-audit-async.sh

```diff
@@ -145,6 +145,11 @@ echo "[DEBUG] source ${REFERENCE_REPOSITORY}/ci/taos/config/config-plugins-audit
 message="Trigger: queued. There are other build jobs and we need to wait.. The commit number is $input_commit."
 cibot_pr_report $TOKEN "pending" "(INFO)TAOS/pr-audit-all" "$message" "${CISERVER}${PRJ_REPO_UPSTREAM}/ci/${dir_commit}/" "$GITHUB_WEBHOOK_API/statuses/$input_commit"
 
+if [[ ${nnstreamer_apptest} == 1 ]]; then
```

Comment:
You give me a homework. :) I start thinking on how to improve the core engine (checker-pr-audit-async.sh) to do not modify  in case of any situation.

Reply:
@ohsewon Any idea?

Reply:
Can the shell script command be run asynchronously?